apps/web/pages/api/auth/[...nextauth].tsx

```diff
@@ -141,15 +154,50 @@ if (isSAMLLoginEnabled) {
   });
 }
 
+if (true) {
```

Comment:
This has no point, as its always being executed

Reply:
I believe it's in order to remind us that we might want to disable it in some instances in the future or allow to config it? not sure!

Reply:
I just liked to encapsulate the provider logic, also as @agustif we can disable easily maybe with an `env` variable.